app/tests/test_models.py

```diff
@@ -89,6 +89,10 @@ def setUp(self):
           .filter(models.User.email=='sly@stone-less-knowledgeable.com')\
           .one()
 
+    def test_get_most_complete_profiles_works(self):
+        users = models.User.get_most_complete_profiles()
+        self.assertEqual(users[-1].email, 'sly@stone-less-knowledgeable.com')
```

Comment:
Can we add another test to make sure we only look inside our current deployment?